learn/inner_workings/known_limitations.mdx

```diff
@@ -119,6 +119,12 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 
 **Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On 64-bit Linux, this limit is approximately 80TiB.
 
+## Maximum task database size
+
+**Limitation:** Meilisearch supports a maximum task dabatase size of 10GiB.
+
+**Explanation:** Depending on your setup, 10GiB should correspond to 5M to 15M tasks. Once the task database is storing over 1M entries, Meilisearch tries to automatically delete finished tasks while continuing to enqueue new tasks as usual. This ensures the task database does not use an excessive amount of resources. If your database reaches the 10GiB limit, Meilisearch will log a warning indicating the engine is not working properly and refuse to enqueue new tasks.
```

Comment:
Not really important, but I think it could reassure the users to know that they should not have to worry at all about reaching this limit because the autodeletion of tasks will kick in way before 

```suggestion
**Explanation:** Depending on your setup, 10GiB should correspond to 5M to 15M tasks. Once the task database stores over 1M entries (roughly 1GiB on average), Meilisearch tries to automatically delete finished tasks while continuing to enqueue new tasks as usual. This ensures the task database does not use an excessive amount of resources. If your database reaches the 10GiB limit, Meilisearch will log a warning indicating the engine is not working properly and refuse to enqueue new tasks.
```